src/main.rs

```diff
@@ -21,7 +16,6 @@ use hyper::service::{ make_service_fn, service_fn };
 use hyper_tls::HttpsConnector;
 
 type HyperClient = Client<HttpsConnector<HttpConnector>>;
-type S3Arc = Arc<dyn rusoto_s3::S3 + Send + Sync>;
```

Comment:
Removed the Arc because the AWS client is itself just a wrapper around an Arc. [client.rs](https://raw.githubusercontent.com/awslabs/aws-sdk-rust/main/sdk/s3/src/client.rs)